diff --git a/tools/idf_tools.py b/tools/idf_tools.py
index 74dce94bb0..b1ef9ac718 100755
--- a/tools/idf_tools.py
+++ b/tools/idf_tools.py
@@ -1570,6 +1570,7 @@ def get_constraints(idf_version):  # type: (str) -> str
 
 
 def action_install_python_env(args):  # type: ignore
+    return
     use_constraints = not args.no_constraints
     reinstall = args.reinstall
     idf_python_env_path, _, virtualenv_python, idf_version = get_python_env_path()
